### ISSUE TITLE
[L10n] The description from the "Sign out from FxA account" prompt doesn't indicate that it is scrollable

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#2525]
**[Notes]:**
- This issue can mislead or the whole information is not provided to the end users since the part of the description that is not visible can be missed.

**[Affected Versions]:**
- Firefox Reality version 8-33550026 (933fcd51)

**[Affected Platforms]:**
- Oculus Quest - Android 7.1.1
- HTC Vive Focus Plus - Android 7.1.2

**[Prerequisites]:**
- The headset is turned on and the browser is opened.
- The app's language is set in "Francais".
- You are logged into an FxA account.

**[Steps to reproduce]:**
1. Click the Settings button.
2. Click the "Manage Account" card.
3. Click the "Sign Out" button.
4. Observe the description from the middle of the prompt.

**[Expected results]:**
- The description fits the prompt's view.
OR
- The description indicates that is scrollable.

**[Actual results]:**
- The description doesn't indicate that is scrollable and doesn't fits the prompt's view.

**[Additional Notes]:**
- The locales on which this issue is reproducible are: NL, FR, DK, IT and RU.
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=1sXyR_vRcdi_kzUeOsfQrNTE7REzV87BR).

**Comment by keianhzo** [comment]
It should not be scrollable. Fixed by https://github.com/MozillaReality/FirefoxReality/pull/2581

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and is no longer reproducible with the latest Firefox Reality version 8-40040048 (ef08ff15) build. The description from the middle of the prompt is now rendered correctly on the affected locales that were mentioned in this issue.
Tested on Oculus Go (Android 7.1.2), Oculus Quest (Android 7.1.1), HTC Vive Focus Plus (Android 7.1.2) and HTC Vive Focus (Android 7.1.2).
